### DISCUSSION TITLE
Grid builder "stack at" not set

### DISCUSSION THREAD

**Discussion by mystifiedmunchkin** [awcodes/filament-tiptap-editor#318] [Q&A] [answered]
Hi everyone. How do you allow the grid divs to stack at the selected width in the grid builder? (sm, md, or lg). I've done it in css but I imagine I might be missing a config file or something for achieving this automatically...

**Comment by awcodes** [comment, marked as answer]
Sounds like you didn't fully follow the install instructions and include the views and css from the plugin in a custom theme.

**Comment by mystifiedmunchkin** [comment]
Thanks btw, fixed it 😊
